.flake8

```diff
@@ -1,4 +1,5 @@
 [flake8]
+max-line-length = 79
```

Comment:
This looks like the max is still 79?

Reply:
Yes, I changed it back following discussion in other channels.